packages/api/src/command/webhook/tenovi.ts

```diff
@@ -27,14 +27,15 @@ import {
   processRequest,
 } from "./webhook";
 import { createWebhookRequest } from "./webhook-request";
+import stringify from "json-stringify-safe";
 
 /**
- * Processes a Tenovi Measurement webhook
+ * Processes a Tenovi Measurement webhook, maps the data, and sends it to the CX
  *
  * @param data Tenovi Measurement webhook
  */
 export const processMeasurementData = async (data: TenoviMeasurement): Promise<void> => {
-  console.log(`Starting to process a Tenovi webhook: ${JSON.stringify(data)}`);
+  console.log(`Starting to process a Tenovi webhook: ${String(data)}`);
```

Comment:
Curious, does it work - result in a string representation of the object? I'd think we need stringify for this since we don't implement `toString()` - maybe Zod does it out-of-the-box?

Reply:
Ah, I think I misunderstood your previous comments on this! Fixed now